constants/battle_anim_constants.asm

```diff
@@ -25,6 +25,7 @@
 	const BATTLEANIMSTRUCT_16
 	const BATTLEANIMSTRUCT_17
 BATTLEANIMSTRUCT_LENGTH EQU const_value
+NUM_ANIM_OBJECTS EQUS "(wActiveAnimObjectsEnd - wActiveAnimObjects) / BATTLEANIMSTRUCT_LENGTH"
```

Comment:
`NUM_OBJECT_STRUCTS`, `NUM_SPRITE_ANIM_STRUCTS`, `MONS_PER_BOX`, `NUM_HOF_TEAMS`, and most other constants just hard-code a number. I'm not sure which approach is better, but can they be made consistent, one way or the other?

Reply:
The downside of this approach is more overhead for the linker. I'm not sure which is better, so I don't know, but thanks for pointing it out.